### PR TITLE
Added the details for the Java Variables and Modifiers task

### PR DESCRIPTION
**Pull request by AndrewSecu**

### CODE REVIEW

File: src/main/java/com/andreisecuqa/accessmodifiers/ChisinauEvidenceCatalog.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        System.out.println(person.getAge());
+
+
+        if (ion001.isRetired) {
+            System.out.println(ion001.firstName + " este pensionat");
+        } else {
+            System.out.println(ion001.firstName + " nu este pensionat");
+        }
+
+//        if (olga001.isRetired) {
```
Comment (AndrewSecu) [inline src/main/java/com/andreisecuqa/accessmodifiers/ChisinauEvidenceCatalog.java]: sterge asta

**Review by busuiocion — CHANGES_REQUESTED** [review]
Sterge tot.

File: src/main/java/com/andreisecuqa/accessmodifiers/ChisinauEvidenceCatalog.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        System.out.println(ion001.getAge());
+
+        Person olga001 = new Person("Olga", "Grozavu", 5646546, 18, false);
+        System.out.println(olga001.getAge());
+
+        Person person = new Person("Valoare", "Valoare", 546465, 524, true);
+        System.out.println(person.getAge());
+
+
+        if (ion001.isRetired) {
```
Comment (AndrewSecu) [inline src/main/java/com/andreisecuqa/accessmodifiers/ChisinauEvidenceCatalog.java]: comenteaza aceasta sectiune

**Review by busuiocion — APPROVED** [review]
Done

**Pull request merged**